jepsen/src/jepsen/control.clj

```diff
@@ -297,19 +298,19 @@
   "Wraps clj-ssh-session in a wrapper for reconnection."
   [host]
   (rc/open!
-   (rc/wrapper {:open    (if *dummy*
-                           (fn [] [:dummy host])
-                           (fn [] (try
-                                    (clj-ssh-session host)
-                                    (catch com.jcraft.jsch.JSchException e
-                                      (error "Error opening SSH session. Verify username, password, and node hostnames are correct.\nSSH configuration is:\n"
-                                             (util/pprint-str (binding [*host* host] (debug-data))))
```

Comment:
I removed the console error print statements, because we're directly passing that data up to the test. I'm OK with keeping it in though.

Reply:
When you say "directly passing up to", do you mean it's thrown as a part of exceptions?

Reply:
yes, sorry! i think it follows a similar function as the printed warnings, so it felt appropriate to remove them.